version.sbt

```diff
@@ -1 +1 @@
-version in ThisBuild := "0.6.1-SNAPSHOT"
+ThisBuild / version := "0.7.0-SNAPSHOT"
```

Comment:
AFAIK, and I just witnessed it yesterday, the sbt release plugin will revert this to "version in ThisBuild" when it runs, so one might skip this instance (or just let it happen naturally).  All the updates elsewhere are appreciated, though.

Reply:
I've actually upgraded the version of the release plugin In this PR.  I wonder if it will still revert...

Reply:
I *think* the absolute newest is being used in processors now and it does revert there.